ddtrace/contrib/requests/connection.py

```diff
@@ -81,8 +79,11 @@ def _wrap_send(func, instance, args, kwargs):
     if not request:
         return func(*args, **kwargs)
 
-    url = request.url
-    hostname = _extract_hostname(url)
+    url = trace_utils._sanitized_url(request.url)
+    method = ""
+    if request.method is not None:
+        method = request.method.upper()
+    hostname, path = _extract_hostname_and_path(url)
```

Comment:
```suggestion
    hostname, path = _extract_hostname_and_path(url)
    # remove integers from path, this will reduce the cardinality of resource names
    route = [ "?" if p.isdigit() else p for p in path.split("/")]
    path = "/".join(route)
```

Reply:
Including the full path here will generate very high cardinality resource names.

@majorgreys Thoughts on removing integers from the path? Do think this approach can lead to any weird edge cases?

Reply:
I feel like integers and uuids are quite common path in parameters.

Reply:
I don't addressing the issue of high cardinality is necessary in this change.

A library user can configure the Agent either with [http resource obfuscation](https://docs.datadoghq.com/tracing/configure_data_security/?tab=http#agent) or more generally with [replace tags](https://docs.datadoghq.com/tracing/configure_data_security/?tab=http#replace-tags).